src/modules/vtol_att_control/vtol_type.cpp

```diff
@@ -381,3 +383,85 @@ bool VtolType::is_channel_set(const int channel, const int target)
 
 	return (channel_bitmap >> channel) & 1;
 }
+
+
+void VtolType::pusher_assist()
+{
+
+	// normalized pusher throttle (standard VTOL) or tilt (tiltrotor), initialize to 0
+	_hover_pusher_or_tilt_forward_actuation = 0.0f;
+
+	// if the thrust scale param is zero or the drone is on manual mode,
```

Comment:
@sfuhrer The comment does not match reality I think.

Reply:
changed to "not in position or altitude control mode", and thus also change the logic to enable it in altitude control mode.